packages/eslint-plugin-svelte/tests/utils/utils.ts

```diff
@@ -119,14 +119,14 @@ export function loadTestCases(
 		.filter(filter)
 		.map((inputFile) => getConfig(ruleName, inputFile));
 
-	const fixable = plugin.rules[ruleName].meta.fixable != null;
+	const fixable = plugin.rules[ruleName].meta?.fixable != null;
 
 	const invalid = listupInput(invalidFixtureRoot)
 		.filter(filter)
 		.map((inputFile) => {
 			const config = getConfig(ruleName, inputFile);
-			const errorFile = inputFile.replace(/input\.[a-z]+$/u, 'errors.yaml');
-			const outputFile = inputFile.replace(/input\.[a-z]+$/u, 'output.svelte');
+			const errorFile = inputFile.replace(/(input|\+.+)\.[a-z]+$/u, 'errors.yaml');
```

Comment:
I need to support SvelteKit route components (e.g. `+page.svelte`) as input.
So I modified the file.